docs/src/content/docs/challenges/angular/55-back-button-navigation.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 🟠 Back-Button-Navigation
+description: Challenge 55 is about overriding browser back button navigation
+author: Ioannis-Tsironis
+contributors:
+  - tomalaforge
+challengeNumber: 55
+command: angular-back-button-navigation
+sidebar:
+  order: 123
+  badge: New
+---
+
+## Information
+
+The goal of this challenge is to override the default behavior of the browser back button in Angular applications.
+
+We have been prompted by the team's PO to provide a specific implementation when displaying dialog components and
+native browser back button is clicked. Currently, the Angular's default behavior when the native back button is clicked is
+to remove the current history entry and go back to previous visited route.
+
+The initial state of the application performs as following:
+When any dialog is displayed and back button is clicked, any opened dialog is closed and app redirects to previous page.
```

Comment:
Replace with this: `When any dialog is displayed and the back button is clicked, any opened dialog is closed, and the app redirects to the previous page.`